src/api/goods.ts

```diff
@@ -6,6 +6,10 @@ export function getAll(): Promise<Good[]> {
     .then(response => response.json());
 }
 
-export const get5First = () => {};
+export const get5First = (goods: Good[]) => {
+  return goods.slice(0, 5);
+};
```

Comment:
You can do as shown below and you will get 5 users:
```suggestion
export const get5First = () => {
  return getAll()
    .then(users => users.sort((a, b) => a.name.localeCompare(b.name))
      .slice(0, 5));
};
```